### ISSUE TITLE
BUG: Controller issue with The Amazing Spider-Man [BLES01547] - Controller vibrates non stop in game.

### ISSUE THREAD

**Issue by Bobble80** [RPCS3/rpcs3#15028]
### Quick summary

In the The Amazing Spider Man (BLES01547 version), as soon as the game boots up the rumble motors on my controller immediately activate and do not stop vibrating at all during gameplay.
I have tried the built in controller on my ROG Ally and also and external bluetooth Xbox controller and the problem happens on both of them. As soon as I quit the game and return back to the RPCS3 menu, the controller vibration stops.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/13875829/RPCS3.log.gz)

### Other details

This exact same issue is also happening with The Amazing Spider-Man 2 - [BLUS41044]

**Comment by antoniodesousa** [comment]
I have the same issue. The controller vibrates non-stop as soon as the game boots up.
[RPCS3.zip](https://github.com/user-attachments/files/15948038/RPCS3.zip)

**Comment by digant73** [comment]
try with the current RPCS3 version. I had a similar issue with some DS3 controllers on killzone 3 and the issue was recently fixed

**Comment by antoniodesousa** [comment]
Nope, it didn't fix it for me. I'm using a Xbox Series controller and the issue only seems to affect the large motor.

**Comment by Bobble80** [comment]
> Nope, it didn't fix it for me. I'm using a Xbox Series controller and the issue only seems to affect the large motor.

Yes, I can confirm it was happening on both an external Xbox controller and the built in controller when I was testing the game on my ROG Ally. 

I’ve no Ally at the moment so I can’t test that RPCS3 update that just came out, so I’ve no idea if the issue was fixed or not.

**Comment by ankur20us** [comment]
I have disabled the vibration and its ok, to play the game without vibration rather controller being vibrating whole time. 

here is the ss.

![image](https://github.com/user-attachments/assets/0d0cb4ff-63ea-424f-90cb-1facc17c1d65)

**Comment by Bobble80** [comment]
Vibration issue is still happening in both TASM games in the current builds of RPCS3 over a year later when the bug report was originally reported. Just a FYI if anyone’s wondering if it’s been fixed or not.

**Comment by 98n00b** [comment]
> ### Resumo rápido
> No The Amazing Spider Man (versão BLES01547), assim que o jogo inicializa, os motores de vibração no meu controle são ativados imediatamente e não param de vibrar durante o jogo. Eu tentei o controle integrado no meu ROG Ally e também um controle externo bluetooth do Xbox e o problema acontece em ambos. Assim que eu saio do jogo e retorno ao menu RPCS3, a vibração do controle para.
> 
> ### Detalhes
> _Nenhuma resposta_
> 
> ### Anexar um arquivo de log
> [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/13875829/RPCS3.log.gz)
> 
> ### Anexe arquivos de captura para problemas visuais
> _Nenhuma resposta_
> 
> ### Configuração do sistema
> _Nenhuma resposta_
> 
> ### Outros detalhes
> Esse mesmo problema também está acontecendo com The Amazing Spider-Man 2 - [BLUS41044]

Hi, I solved this problem by configuring player 2, changing the handlers option to "null". On my device, both player 1 and player 2 were configured for the same joystick.

**Comment by digant73** [comment]
@Bobble80 I see some users reported the issue with DualSense handler. Are you having the issue using SDL handler?

**Comment by Bobble80** [comment]
> [@Bobble80](https://github.com/Bobble80) I see some users reported the issue with DualSense handler. Are you having the issue using SDL handler?

Yeah, I’m using the SDL handler. It’s been an issue for the last few years now.

**Comment by digant73** [comment]
> > [@Bobble80](https://github.com/Bobble80) I see some users reported the issue with DualSense handler. Are you having the issue using SDL handler?
> 
> Yeah, I’m using the SDL handler. It’s been an issue for the last few years now.

are the pads connected via BT (if applicable) or cable?

**Comment by Bobble80** [comment]
> > > [@Bobble80](https://github.com/Bobble80) I see some users reported the issue with DualSense handler. Are you having the issue using SDL handler?
> > 
> > 
> > Yeah, I’m using the SDL handler. It’s been an issue for the last few years now.
> 
> are the pads connected via BT (if applicable) or cable?

Both. I’m mainly using RPCS3 on my ROG Ally so the controller is built into that device of course. And I’ve tried connecting an Xbox controller to the Ally via Bluetooth and that also had the vibration issue going on in both TASM games.

**Comment by digant73** [comment]
@Bobble80 please test if an old version such as 0.0.23-14014 (reported in #7425 as working for some users) is working for you

**Comment by Bobble80** [comment]
> [@Bobble80](https://github.com/Bobble80) please test if an old version such as 0.0.23-14014 (reported in [#7425](https://github.com/RPCS3/rpcs3/issues/7425) as working for some users) is working for you

I just tested that version 0.0.23-14014 now with TASM 1 and the constant vibration issue is still there. I was using the Xinput option in the controller settings.

**Comment by digant73** [comment]
> > [@Bobble80](https://github.com/Bobble80) please test if an old version such as 0.0.23-14014 (reported in [#7425](https://github.com/RPCS3/rpcs3/issues/7425) as working for some users) is working for you
> 
> I just tested that version 0.0.23-14014 now with TASM 1 and the constant vibration issue is still there. I was using the Xinput option in the controller settings.

Ok. So you tried `XInput` and `SDL` handlers in the `Pad` menu and the issue is present with both of them, right?

**Comment by antoniodesousa** [comment]
Still same issue for me. `XInput` or `SDL` makes no difference.
Only workaround is disabling the large motor. The small motor has no issues.

**Comment by Bobble80** [comment]
> > > [@Bobble80](https://github.com/Bobble80) please test if an old version such as 0.0.23-14014 (reported in [#7425](https://github.com/RPCS3/rpcs3/issues/7425) as working for some users) is working for you
> > 
> > 
> > I just tested that version 0.0.23-14014 now with TASM 1 and the constant vibration issue is still there. I was using the Xinput option in the controller settings.
> 
> Ok. So you tried `XInput` and `SDL` handlers in the `Pad` menu and the issue is present with both of them, right?

Yeah, issue is there on both handlers in the pad menu, I’ve tried both. And it’s happening in both version 0.0.23 right up to the current RPCS3 version.

**Comment by digant73** [comment]
Please test if the attached `rpcs3.exe` is working and attach your `rpcs3.log`.
You can use, XInput, SDL, DS3 or Dualsense pad handlers.

[rpcs3.zip](https://github.com/user-attachments/files/22244493/rpcs3.zip)

**Comment by antoniodesousa** [comment]
> Please test if the attached rpcs3.exe is working and attach your rpcs3.log.
> You can use, XInput, SDL, DS3 or Dualsense pad handlers.

Yep, it does fix the issue for me. No more vibrating like crazy.
I also tried the PR https://github.com/RPCS3/rpcs3/pull/17487 and it's fixed there as well.
